### PR TITLE
Document how to use update-bikeshed-cache.sh

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

**Comment by sideshowbarker** [comment]
> thanks @sideshowbarker,
> 
> this assumes one is sitting on the master branch?

Well at least not the gh-pages branch. But it could be done first from a PR branch and then merged into master.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,18 @@ cp -R .spec-data/* ./bikeshed/bikeshed/spec-data
...
+
+For the purpose of giving consistent builds, copies of some Bikeshed data files are stored in this repository and used by the CI build to auto-publish the https://w3c.github.io/webauthn/ draft.
+
+The repository copies of those Bikeshed data files must be periodicallly updated and re-committed. To update and re-commit them, run the following commands:
+
+```
+./update-bikeshed-cache.sh \
+  && git add .spec-data .bikeshed-include \
+  && git commit -m "Bikeshed spec data update" . \
+  && git push
```
Comment (equalsJeffH) [inline README.md:53]:
```suggestion
cd <your local w3c/webauthn repo or fork clone>
git co master
git pull           # if necessary...
git co -b <a new branch name>
./update-bikeshed-cache.sh \
  && git add .spec-data .bikeshed-include \
  && git commit -m "Bikeshed spec data update" . \
  && git push
<in the w3c/webauthn repo create a pull request for this new branch, have it reviewed and merged>
```

**Comment by sideshowbarker** [comment]
> how about this more detailed set of instructions?

👍 looks great

File: README.md [hunk 2]
```diff
@@ -40,6 +40,23 @@ cp -R .spec-data/* ./bikeshed/bikeshed/spec-data
...
 
+# Updating copies of Bikeshed data files stored in this repo
+
+For the purpose of giving consistent builds, copies of some Bikeshed data files are stored in this repository and used by the CI build to auto-publish the https://w3c.github.io/webauthn/ draft.
+
+The repository copies of those Bikeshed data files must be periodicallly updated and re-committed. To update and re-commit them, run the following commands:
+
+```
+cd <your local w3c/webauthn repo or fork clone>
+git co master
```
Comment (equalsJeffH) [inline README.md:51]:
```suggestion
git checkout master
```

File: README.md [hunk 3]
```diff
@@ -40,6 +40,23 @@ cp -R .spec-data/* ./bikeshed/bikeshed/spec-data
...
+git pull           # if necessary...
+git co -b <a new branch name>
```
Comment (equalsJeffH) [inline README.md:53]:
```suggestion
git checkout -b <a new branch name>
```

**Pull request merged**